src/index.js

```diff
@@ -44,6 +44,7 @@ export { default as Icon } from "./Icon";
 export { default as InputField } from "./InputField";
 export { default as InputGroup } from "./InputGroup";
 export { default as InputFile } from "./InputFile";
+export { default as InputStepper } from "./InputStepper";
```

Comment:
You defined it for `src/index.js`, but don't forget to declare the export type in `src/index.js.flow`